core/src/main/java/org/opensearch/sql/expression/datetime/DateTimeFunction.java

```diff
@@ -602,10 +606,14 @@ private DefaultFunctionResolver utc_timestamp() {
    */
   private DefaultFunctionResolver week(BuiltinFunctionName week) {
     return define(week.getName(),
+        implWithProperties((functionProperties, arg) -> DateTimeFunction.weekOfYearToday(new ExprIntegerValue(0),
```

Comment:
you can make new ExprIntegerValue(0) a static constant of this file.  Maybe call it something like `DEFAULT_WEEK_OF_YEAR_MODE`?

Reply:
Fixed in 85c93e677ba64de401fa4b4115acfe428005e003